netbench/netbench-collector/Cargo.toml

```diff
@@ -11,6 +11,7 @@ publish = false
 handlebars = "4"
 netbench = { version = "0.1", path = "../netbench" }
 serde = { version = "1", features = ["derive"] }
-serde_json = "1"
+serde_json = "1.0.96"
```

Comment:
Unless we explicitly need at least this patch version, we prefer to just specific the major version. And if we do explicitly need at least that patch version, we should add a comment explaining why.